.github/workflows/ci.yml

```diff
@@ -73,17 +73,22 @@ jobs:
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
 
       - name: Get Changed Files
-        uses: tj-actions/changed-files@1d6e210c970d01a876fbc6155212d068e79ca584
-        id: changed-files
-        with:
-          separator: ","
+        id: changed
+        continue-on-error: true
+        run: |
+          echo "CHANGED_FILES<<EOF" >> $GITHUB_ENV
+          gh pr diff ${{ github.event.number }} --name-only | sed -e 's|$|,|' | xargs -i echo "{}" >> $GITHUB_ENV
```

Comment:
_Probably_ not important, but how well does this handle file names containing newline characters or commas?

You can create one with:

```bash
touch $'hello\nworld'
```

Reply:
This won't handle them. It's probably safe to assume that those files won't be added, though.